wremnants/syst_tools.py

```diff
@@ -318,6 +320,16 @@ def add_pdf_hists(results, df, dataset, axes, cols, pdfs, base_name="nominal", a
         else:
             pdfHist = df.HistoBoost(pdfHistName, axes, [*cols, tensorName], tensor_axes=[pdf_ax], storage=hist.storage.Double())
             alphaSHist = df.HistoBoost(alphaSHistName, axes, [*cols, tensorASName], tensor_axes=[as_ax], storage=hist.storage.Double())
+
+            if propagateToHelicity:
```

Comment:
What is the difference in this block with respect to the "if addhelicity" block in L313?

Reply:
removed as @cippy clarified that the usage of addHelicity was not what he intended it to be